cmd/tracee-ebpf/internal/printer/printer.go

```diff
@@ -260,6 +260,38 @@ func (p *gobEventPrinter) Print(event trace.Event) {
 	if err != nil {
 		p.Error(err)
 	}
+
```

Comment:
Initially I added Forward output to ebpf just to verify it worked. This provides all the events then before the rules engine gets involved which may be useful in some scenarios but I suspect we should just remove this.